Makefile

```diff
@@ -2,7 +2,6 @@
 fix:
 	wsl --allow-cuddle-declarations --fix ./... && \
 		gci write . && \
-		markdownlint --fix . && \
```

Comment:
removing this since we don't yet enforce a markdown linter, and also our CONTRIBUTING.md has a lot of lint issues, we can reintroduce this once we've addressed all the lint issues